### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by gi8lino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,6 @@ A small cli tool to generate TOTP (Time-based One-time Password) tokens.
 
 ## Usage
 
-```
-Usage of otpgen:
-  -s, --secretkey string   supply a valid TOTP secret key to generate a token from
+```bash
+otpgen -s|--secretkey SECRETKEY
```
Comment (rxbn) [inline README.md]: IMO, we should stick to the help output of pflag library. Why should we change that?

Comment (gi8lino) [reply README.md]: You are absolutely right! I reset the changes.

**Review by rxbn — APPROVED** [review]
Ok

**Comment by rxbn** [comment]
ready to merge, no new release required

**Pull request merged**